src/ripple/rpc/RPCExecutionConcept.svg

```diff
@@ -0,0 +1,46 @@
+@startuml
+
+Title "Request Processing Execution Concept"
+
+participant "Client" as C
+participant "Door asio strand" as Door
+participant "PlainHTTPPeer asio strand" as P
+participant "PlainWSPeer asio strand" as WS
+collections "JobQueue worker thread" as J
+
+C -> Door : Request received
+create P
+Door -> P : Object created
+activate P
+create WS
+P -> WS : Web Socket upgrade
+destroy P
+deactivate P
+activate WS
+WS -> J : Execute handler
+J -> WS : Return handler result
+WS -> C : Send response
+C -> WS : Send next message

```

Comment:
I think this should be "Read next message".

Reply:
Good catch, will change